packages/graphql-yoga/src/index.ts

```diff
@@ -164,16 +168,25 @@ export class GraphQLServer extends BaseNodeGraphQLServer {
       headers,
     })
 
-    // In failed requests res.json() will throw an error
-    // So we just parse the body ourselves
-    const body = JSON.parse(res.body)
-
-    return {
-      statusCode: res.statusCode,
-      headers: res.headers,
-      data: body?.data as TData,
-      errors: body?.errors as Error[],
-    }
+    return new Response(
```

Comment:
Is this the only place `cross-undici-fetch` is required and used for? So I would suggest that we just provide fastify inject response. Unless this is required by W3C helix as a peer dep